web/packages/teleterm/src/sharedProcess/ptyHost/ptyProcess.ts

```diff
@@ -75,6 +75,9 @@ export class PtyProcess extends EventEmitter implements IPtyProcess {
 
     this._setStatus('open');
     this.emit(TermEventEnum.Open);
+    if (this.options.helpMsg) {
+      this.emit(TermEventEnum.Data, this.options.helpMsg);
```

Comment:
Let's explain how we want this to be displayed in terminal emulator in the app and circumvent the underlying PTY at the same time.